src/components/scenes/StakingChangeScene.js

```diff
@@ -221,7 +223,7 @@ export const StakingChangeSceneComponent = (props: Props) => {
   const sliderDisabled = tx == null || amount == null || amount === '0' || error != null
 
   const renderAdd = () => {
-    const apyValue = sprintf(s.strings.staking_estimated_return, `${apy}%`)
+    const apyValue = sprintf(maxApy != null && apy === maxApy ? s.strings.staking_estimated_return_up_to : s.strings.staking_estimated_return, `${apy}%`)
```

Comment:
If `apy` is never nullish, then you can shorten it:

```js
sprintf(apy === maxApy ? s.strings.staking_estimated_return_up_to : s.strings.staking_estimated_return, `${apy}%`)
```

But if `apy` is ever nullish, then why do we always allow the string interpolation?